.gitignore

```diff
@@ -13,6 +13,7 @@ command-log.csv
 experimental_console.py
 stress.sh
 test.py
+Scripts/test_plots.py
```

Comment:
Let's use a lower-case `experiments` folder and just ignore everything inside it.

We reserver `scripts` for stuff that's actually part of wright.